### PR TITLE
MINOR: enable DEBUG logging for all tests by default

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
I think we should enable DEBUG logging for test by default. Often if a test fails, we have very limited information why.

File: src/test/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=DEBUG, stdout
```
Comment (jaceklaskowski) [inline src/test/resources/log4j.properties]: Shouldn't `stdout` have the corresponding `log4j.appender.stdout.target=System.out`?

Comment (mjsax) [reply src/test/resources/log4j.properties]: I guess the name is just a little confusion... we want to capture debug-level logs, not what's written to System.out and print them to the console/stdout...

Or do I miss understand your comment?

Comment (jaceklaskowski) [reply src/test/resources/log4j.properties]: I'm just referring to the name as is that has no corresponding `appender` defined whether it's `System.out` or not. I thought it's required to have a `log4j.appender.[name].target` entry defined.

Comment (mjsax) [reply src/test/resources/log4j.properties]: Default target is `System.out` -- no need to specify.

**Pull request merged**